### ISSUE TITLE
Requesting crafting a crusher from Factorium caused a crash

### ISSUE THREAD

**Issue by AlexiyOrlov** [CyclopsMC/IntegratedCrafting#92]
#### Issue type:

- :bug: Bug 

____

#### Short description:

I tried to request crafting a Crusher from Factorium and the game crashed.

#### Steps to reproduce the problem:

1. Set up recipes for the crusher
2. Request crafting

#### Expected behaviour:

Crusher to be crafted
____

#### Versions:

- This mod: 1.19.3-1.1.0
- Minecraft: 1.19.3
- Forge: 44.1.17

#### Log file:

https://pastebin.com/wpDdNYwT

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by AlexiyOrlov** [comment]
Hmm, I can't craft anything from that mod

**Comment by rubensworks** [comment]
Is there a vanilla-only recipe using which you can reproduce the bug?

**Comment by AlexiyOrlov** [comment]
No, vanilla recipes work fine

**Comment by rubensworks** [comment]
Could you give some details about this Crusher recipe, and how it is crafted exactly?

**Comment by AlexiyOrlov** [comment]
It's from this mod [link](https://www.curseforge.com/minecraft/mc-mods/factorium). Also today I tried crafting items from AE2, and the same crash occured, so it might not be a mod-specific problem. After that crash I opened the storage terminal and no items showed up from the network. I tried breaking it and placing again, but it didn't fix. Very unfortunate. Also I noticed this in the log: [paste](https://pastebin.com/Gdb5SDvd)
